include.js

```diff
@@ -7,6 +7,12 @@ let cards = [
     githubLink: 'https://github.com/aniketmdinde'
   },
   {
+    artName 'Loading Dot Animation',
+    pageLink '.Artahmedrazaindex.html',
+    imageLink '.ArtahmedrazaLoading-Dot.gif',
+    author 'Ahmed raza',
+    githubLink 'httpsgithub.comahmedrazabaloch'
+ },
```

Comment:
# Feedback - Invalid JS Object
- [ ] missing colon (`:`) between the key and the value
- [ ] missing curly next curly braces for the object bellow
- [ ] incorrect paths/links for [ pageLink, imageLink, githubLink ] 
those seems to miss all slashes (`/`)
Please unsure those are correct before pushing your next commit(s)

## Context 
```js
 {
    artName 'Loading Dot Animation',
    pageLink '.Artahmedrazaindex.html',
    imageLink '.ArtahmedrazaLoading-Dot.gif',
    author 'Ahmed raza',
    githubLink 'httpsgithub.comahmedrazabaloch'
 },
   artName: 'LoopLamina',
```